api/v1alpha3/zz_generated.conversion.go

```diff
@@ -826,9 +826,11 @@ func autoConvert_v1alpha4_MachineDeploymentStatus_To_v1alpha3_MachineDeploymentS
 	out.AvailableReplicas = in.AvailableReplicas
 	out.UnavailableReplicas = in.UnavailableReplicas
 	out.Phase = in.Phase
+	// WARNING: in.Conditions requires manual conversion: does not exist in peer-type
 	return nil
 }
 
+// FIXME(awander): This func is removed after running `make generate`. I assume b/c there is no Conditions fields in v1alpha3 for MachineDeploymentStatus. What is the correct way to handle this?
```

Comment:
Can someone help me here. What is the correct way to handle this?

Reply:
If I got this right you should make sure to get roundtrip conversion by using marshal and unamarshal to in an annotation like we already have for `spec.Strategy` in https://github.com/kubernetes-sigs/cluster-api/blob/440a0035c0ddf463147652976878cc2015dbc235/api/v1alpha3/conversion.go#L119-L159